app/controllers/candidate_interface/new_references/request_reference/review_controller.rb

```diff
@@ -0,0 +1,30 @@
+module CandidateInterface
+  module NewReferences
+    class RequestReference::ReviewController < ReviewController
+      include RequestReferenceOfferDashboard
+      before_action :set_reference, :set_policy
+      before_action :verify_reference_can_be_requested, only: %i[feedback_requested]
+
+      def feedback_requested
```

Comment:
would `request_feedback` be a more accurate name?

Reply:
Done! ♻️